CHANGELOG.md

```diff
@@ -13,12 +14,65 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
+# r1.3
+## Release Notes
+
+This **pre-release** contains the definition and documentation of
+* Brand Registration v0.1.0
+* Verified Caller v0.1.0
+
+The API definition(s) are based on
+* Commonalities r3.2
+* Identity and Consent Management r3.2
+
+## Brand Registration v0.1.0
+This is the first initial version of Brand Registration API.
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/VerifiedCaller/r1.3/code/API_definitions/brand-registration.yaml&nocors)
+  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/VerifiedCaller/r1.3/code/API_definitions/brand-registration.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/VerifiedCaller/blob/r1.3/code/API_definitions/brand-registration.yaml)
+
+### Added
+* N/A
+
+### Changed
+* N/A
+
+### Fixed
+* N/A
+
+### Removed
+* N/A
+
+## Verified Caller v0.1.0
+This is the first initial version of Verified Caller API
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/VerifiedCaller/r1.3/code/API_definitions/verified-caller.yaml&nocors)
+  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/VerifiedCaller/r1.3/code/API_definitions/verified-caller.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/VerifiedCaller/blob/r1.3/code/API_definitions/verified-caller.yaml)
+
+### Added
+* N/A
+
+### Changed
+* N/A
+
+### Fixed
+* Remove occurrences of "Pre-Announce" from API name throughout the repository by @alpaycetin74 in https://github.com/camaraproject/VerifiedCaller/pull/61
+
+### Removed
+* N/A
+
+**Full Changelog**: https://github.com/camaraproject/VerifiedCaller/compare/r1.1...r1.3
```

Comment:
As this is the first public version, the changelog should just list all commits that are part of the release. For future releases, then a comparison with the previous release would be given.
```suggestion
**Full Changelog**: https://github.com/camaraproject/VerifiedCaller/commits/r1.3
```